.github/workflows/python-package.yml

```diff
@@ -40,5 +41,30 @@ jobs:
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
     - name: Test with pytest and coverage
       run: |
-        coverage run -m pytest
-        coverage report
+        pytest --cov=lobsterpy --cov-append --splits 10 --group ${{ matrix.split }} --durations-path ./lobsterpy/.pytest-split-durations
```

Comment:
To me it looks like you are doing 10 splits but only run 5 out of the 10? Do I overlook something?